src/components/left-nav-panel.sass

```diff
@@ -72,6 +73,17 @@
   .button:hover
     border-color: $button-border-hover-color
 
+  .footer
+    flex-grow: 0
+    flex-shrink: 0
+    flex-basis: $left-nav-footer-height
```

Comment:
This looks good - I've just used `height` in the past but it seems like this is more specific. Is it necessary to set `flex-grow` and `shrink` here?

Reply:
It seemed like the more "flexy" way to specify a constant piece within a set of flexible containers.  So we define the start dimension with `flex-basis` and then set `flex-grow` and `flex-shrink` to 0 to tell it not to change size.  It's verbose and basically the same as `height`, but I like using the flex properties to specify dimensions if we're using flex boxes.

Reply:
Oh I just meant, the defaults on `flex-grow` and `flex-shrink` are already 0, right?

Reply:
Ah, good point!

Reply:
Looks like flex-shrink is 1 by default:
https://developer.mozilla.org/en-US/docs/Web/CSS/flex-shrink
Flex-grow is 0:
https://developer.mozilla.org/en-US/docs/Web/CSS/flex-grow

Reply:
We also need to worry about inherited values since this layout does use a number of flex boxes.